engine/core/src/main/java/org/datacleaner/util/sort/SortMergeWriter.java

```diff
@@ -174,6 +177,28 @@ public File write(String filename) {
     }
 
     /**
+     * @param resource
+     * @return the written count of rows
+     */
+    public int write(final Resource file) {
+        W writer = null;
+        ObjectInputStream[] tempFileObjectInputStreams = null;
+        try {
+            writer = createWriter(file);
+            return processRows(writer, tempFileObjectInputStreams);
```

Comment:
I don't get this. The way I see it 'tempFileObjectInputStreams' will always be null here, and also in the write(File) method below.

I would imagine that when we add a write(Resource) method, then we can implement the write(File) method simply by calling write(new FileResource(file)) and then reuse the complete write method code that way.